core/src/main/java/dev/streamx/cli/ingestion/publish/PublishCommand.java

```diff
@@ -42,12 +56,44 @@ protected String getChannel() {
   @Override
   protected void perform(Publisher<JsonNode> publisher) throws StreamxClientException {
     List<PayloadArgument> mergedPayloadArgumentList = prependPayloadFile();
-    JsonNode jsonNode = payloadResolver.createPayload(mergedPayloadArgumentList);
-    PublisherSuccessResult result = publisher.publish(publishTargetArguments.getKey(), jsonNode);
+    JsonNode payload = payloadResolver.createPayload(mergedPayloadArgumentList);
+    JsonNode schemaJson = null;
```

Comment:
dej to do metody wyliczanie tego `schemaJson`

Reply:
done